### PR TITLE
DOC-3083: Update SFD to indicate discussions available for edx.org learners

### PR DESCRIPTION
**Pull request by catong**
DOC-3083 This PR corrects the learners' guides to indicate that discussions are available on the edx mobile app.
- [x]  @lamagnifica Quick check please
- [x] Run tests cleanly
- [x] Squash commits

### CODE REVIEW

File: en_us/shared/students/SFD_mobile.rst [hunk 1]
```diff
@@ -15,22 +15,16 @@ smartphone or an iPhone to take edX courses.
 Overview of the edX Mobile App
 ********************************************************
 
-.. The following paragraph describes the features of the edX mobile app for partners/edx.org (video only)
-.. Alison, DOC-1840, June 2015
-
```
Comment (lamagnifica) [inline en_us/shared/students/SFD_mobile.rst:20]: why remove the description of the "only" directive?

Comment (lamagnifica) [inline en_us/shared/students/SFD_mobile.rst:20]: how about this: put the sentence "The edX mobile apps are companions to the `edx.org`_ website." under the only:: Partners directive. Then, new paragraph shared to both partners and open, all the rest (line 20-26 and 30-35 look identical to me)

File: en_us/shared/students/SFD_mobile.rst [hunk 2]
```diff
@@ -107,24 +101,11 @@ on a computer.
...
-  Right now, you cannot use the edX mobile apps to participate in course
-  discussions. To read or contribute to the discussions, use a web browser on
-  a computer.
-
-.. The following paragraph describes the features of the edX mobile app for Open edX (adds notifications, assessments, discussions)
-.. Alison, DOC-1840, June 2015
-
-.. only:: Open_edX
-
   You can read and contribute to course discussions in the edX mobile apps
```
Comment (lamagnifica) [inline en_us/shared/students/SFD_mobile.rst]: remove indent for this para

**Comment by lamagnifica** [comment]
one suggestion to streamline/consolidate what used to be in two separate "only" directives

**Comment by lamagnifica** [comment]
👍 :)

**Comment by catong** [comment]
These mobile topics need a lot of further work to be correctly conditionalized! I will make some stories.

**Comment by catong** [comment]
Merging https://github.com/edx/edx-documentation/pull/1101

**Pull request merged**